main.ipynb

```diff
@@ -710,10 +710,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "id": "71f3bf15-c584-44da-a789-c821acb3f9a6",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Idx 0: Waypoint(x=np.float64(0.0), y=np.float64(0.0), z=30.48, speed=0.0)\n",
```

Comment:
Can you fix the creation of waypoints so that x and y are primitive floats instead of `np.float`?